### PR TITLE
#4049 Add a test to verify links of stream, connector and topic can be worked normally

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4049)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/configs/cypress.e2e.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "integrationFolder": "cypress/e2e",
   "reporterOptions": {
     "mochaFile": "../test-reports/e2eReport-[hash].xml",
-    "toConsole": true
+    "toConsole": true,
+    "jenkinsMode": true
```
Comment (konekoya) [inline ohara-manager/client/cypress/configs/cypress.e2e.json]: 👍

File: ohara-manager/client/cypress/e2e/pipeline.test.js [hunk 2]
```diff
@@ -313,6 +335,7 @@ describe('Pipeline Page', () => {
     cy.reload();
 
     // 1. cannot create link to a source
+    cy.log(`cannot create link for perf source -> ftp source`);
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/pipeline.test.js]: How about _cannot create a link from a perf source to a ftp source_ ?

File: ohara-manager/client/cypress/e2e/pipeline.test.js [hunk 3]
```diff
@@ -78,6 +80,7 @@ Cypress.Commands.add('addElement', (name, kind, className) => {
         .click();
     } else if (kind === KIND.topic) {
       topics.push(name);
+      cy.log(`current all topics in pipeline: ${topics.join(',')}`);
```
Comment (konekoya) [inline ohara-manager/client/cypress/e2e/pipeline.test.js]: _Available topics in the pipeline_ or _All topics that are currently available in the pipeline_ :)

**Comment by saivirtue** [comment]
retry manager-e2e

**Comment by saivirtue** [comment]
retry manager-e2e

**Pull request merged**